application_cluster/variables.tf

```diff
@@ -88,3 +88,43 @@ variable application_cluster_target_group_protocol {
   type = "string"
   default = "HTTP"
 }
+
+variable "healthcheck_interval" {
+  description = "see https://www.terraform.io/docs/providers/aws/r/lb_target_group.html#interval"
+  default = 30
+}
+
+variable "healthcheck_path" {
+  description = "see https://www.terraform.io/docs/providers/aws/r/lb_target_group.html#path"
+  default = "/"
```

Comment:
Problematic because there are differents between application- and network-loadbalancer (see https://www.terraform.io/docs/providers/aws/r/lb_target_group.html#path)

Anybody have a suggestion?

Reply:
is there a possibility to work with count on a variable that determines which kind of LB is used. So the resources for Network-LB are only build if necessary and vice versa?

Reply:
Not in the variables file itself but it would be possible to define a different target group by different load balancer types ... but i don't really like ist